### ISSUE TITLE
Change error message when RE name is too long to more user friendly

### ISSUE THREAD

**Issue by Maladie** [kyma-project/kyma#1204]
**Description**

When Remote Environment name exceeds 253 characters, the error message appears as can be seen below. Please, consider changing it to more descriptive and user friendly.

![image](https://user-images.githubusercontent.com/30122892/46735212-db863800-cc95-11e8-9645-7a9f7d245c59.png)

**Comment by lszymik** [comment]
We spotted a couple of potential improvements:

- The validation in UI should be added, and the error message should be human-readable
- The error message indicates that UI is not using UI API layer (graphql) We created a UI API functionality for it and it is providing additional validation
- Moreover, it will be nice to have a better possibility to create tags. It should be possible to finish tag entry by any whitespace. As soon as there is whitespace after value, the next tag should be possible.
